docs/CHANGELOG.asciidoc

```diff
@@ -90,10 +91,356 @@ For detailed information about the n.n.n release, review the following sections.
 
 --
 
+[[release-notes-7.12.0]]
+== {kib} 7.12.0
+
+coming::[7.12.0]
+
+For detailed information about the {kib} 7.12.0 release, review the following information.
```

Comment:
The last line also says:

For detailed information about the 7.12.0 release, refer to Enhancements and bug fixes.

Suggest rewording one of them.